### PR TITLE
remove the boulder summit, add a blog post explaining why

### PR DESCRIPTION
**Pull request by stew**

### CODE REVIEW

**Comment by rossabaker** [comment]
I have seen more good thoughts that would make fine concurring opinions in a personal space.  In the organization's space, I find that this says what needs to be said and no more. :+1:

**Comment by non** [comment]
I agree with @rossabaker

File: _config.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ twitter: typelevel
 email: info@typelevel.org
 gitter: "https://gitter.im/orgs/typelevel/rooms"
 google_groups: "https://groups.google.com/group/typelevel"
-baseurl: http://typelevel.org
+baseurl: /
```
Comment (tpolecat) [inline _config.yml]: Was this change intentional?

**Comment by tpolecat** [comment]
This whole thing is sad, but I agree it's what we should do.

File: _config.yml (hunk 1 above)
Comment (stew) [inline _config.yml]: oh certainly no, thank you

**Comment by milessabin** [comment]
Also :+:.
